### PR TITLE
[IMP] event: change event form view to improve usability

### PR DESCRIPTION
**Pull request by pko-odoo**
PURPOSE

Change the event form view to improve its usability and make it clearer

SPECIFICATION

- Change some stats button icons
- Change order of the fields
- Put the date fields just below the title since it's a mandatory field.
- Convert the note field into an html field.
- Convert seats_availability selection field into seats_limited a boolean field.
- Move 'website_menu', 'website_track' and website_track_proposal' to the customize dropdown menu in the frontend.

LINKS

Task ID : 2198660
Enterprise PR : odoo/enterprise#8954

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website_event/models/event_event.py [hunk 1]
```diff
@@ -88,19 +92,19 @@ def _get_menu_entries(self):
...
+                event.menu_id.unlink()
+            elif event.website_menu and not event.menu_id:
+                root_menu = self.env['website.menu'].create({'name': event.name, 'website_id': event.website_id.id})
+                event.menu_id = root_menu
+                for sequence, (name, url, xml_id) in enumerate(event._get_menu_entries()):
+                    event._create_menu(sequence, name, url, xml_id)
 
     def _create_menu(self, sequence, name, url, xml_id):
         if not url:
+            self.env['ir.ui.view'].search([('name', '=', name + ' ' + self.name)]).unlink()
```
Comment (tde-banana-odoo) [inline addons/website_event/models/event_event.py]: As self.name is translatable, it will probably fail.

Moreover why do you remove a view here ?

Comment (pko-odoo) [reply addons/website_event/models/event_event.py]: Hello @tde-banana-odoo 

Right now, when disabling the website menu and enabling it again after it creates another ir.ui.view as the new_page(ispage=False) method is called. Therefore you get a traceback when clicking on the customize dropdown on the frontend. It's the only way I found to unlink that ir.ui.view.

Comment (tde-banana-odoo) [reply addons/website_event/models/event_event.py]: Indeed it is a bit complicated as I did not find any link to the view ... broken by design.

Ok then, to be cleaned in master post freeze with real links.

**Comment by tde-banana-odoo** [comment]
@robodoo r+ rebase-merge

**Comment by d-fence** [comment]
robodoo retry

**Comment by d-fence** [comment]
robodoo retry

**Comment by tde-banana-odoo** [comment]
@robodoo r+ rebase-merge

**Pull request closed without merging**